### PR TITLE
a bit of template DRYing.

### PR DESCRIPTION
**Pull request by akash1810**

### CODE REVIEW

File: kahuna/public/js/image/view.html [hunk 1]
```diff
@@ -188,66 +188,75 @@
...
+                                Unknown (click ✎ to add)
+                            </span>
                         </span>
-                    </span>
-                </dd>
+                    </dd>
+                </dl>
 
 
-                <dt ng:if="ctrl.metadata.subLocation || ctrl.metadata.city || ctrl.metadata.state || ctrl.metadata.country" class="image-info__group--dl__key metadata-line__key">Location</dt>
```
Comment (akash1810) [inline kahuna/public/js/image/view.html]: This `ng:if` looked a bit crazy!

File: kahuna/public/js/image/view.html [hunk 2]
```diff
@@ -188,66 +188,75 @@
...
-                <dd ng:if="ctrl.metadata.subLocation || ctrl.metadata.city || ctrl.metadata.state || ctrl.metadata.country" class="image-info__group--dl__value metadata-line__info">
-                    <span ng:repeat="prop in ['subLocation', 'city', 'state', 'country']" ng:if="ctrl.metadata[prop]">
-                        <span class="metadata-line__info">
-                            <a ui:sref="search.results({query: (ctrl.metadata[prop] | queryFilter:ctrl.locationFieldMap[prop])})">{{ctrl.metadata[prop]}}</a></span><span ng:if="! $last">,</span>
-                    </span>
-                </dd>
+                <dl class="flex-container full-width"
```
Comment (jamesgorrie) [inline kahuna/public/js/image/view.html]: It seems weird to have a `dl` for each one?
Isn't it ont `dl` with multiple items?

Comment (akash1810) [inline kahuna/public/js/image/view.html]: > Isn't it ont dl with multiple items?

[Yes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl). Only `dt` and `dd` are valid according to the spec. Reverted this.

**Comment by akash1810** [comment]
closing as changes have been rolled into https://github.com/guardian/grid/pull/1368.

**Pull request closed without merging**